### ISSUE TITLE
I cant seem to play/install the 1.12.2 pack on technic launcher

### ISSUE THREAD

**Issue by Whattabeach** [xJon/The-1.12.2-Pack#753]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
**yes**.
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
**yes**.
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
**yes**.

**Information required**
1. [B#707] Starting download of https://mirror.technicpack.net/Technic/version/1.12.2/1.12.2.json, with 3 tries remaining
[B#707] java.io.FileNotFoundException: C:\Users\jenso\AppData\Roaming\.technic\modpacks\the-1122-pack\bin\natives\SAPIWrapper_x64.dll (The process cannot access the file because it is being used by another process)
[B#707] 	at java.io.FileOutputStream.open0(Native Method)
[B#707] 	at java.io.FileOutputStream.open(Unknown Source)
[B#707] 	at java.io.FileOutputStream.<init>(Unknown Source)
[B#707] 	at java.io.FileOutputStream.<init>(Unknown Source)
[B#707] 	at net.technicpack.utilslib.ZipUtils.unzipEntry(ZipUtils.java:76)
[B#707] 	at net.technicpack.utilslib.ZipUtils.unzipFile(ZipUtils.java:136)
[B#707] 	at net.technicpack.launchercore.install.tasks.UnzipFileTask.runTask(UnzipFileTask.java:59)
[B#707] 	at net.technicpack.launchercore.install.tasks.TaskGroup.runTask(TaskGroup.java:85)
[B#707] 	at net.technicpack.launchercore.install.InstallTasksQueue.runAllTasks(InstallTasksQueue.java:49)
[B#707] 	at net.technicpack.launchercore.install.ModpackInstaller.installPack(ModpackInstaller.java:42)
[B#707] 	at net.technicpack.launcher.launch.Installer$1.run(Installer.java:123)
[B#707] 	at java.lang.Thread.run(Unknown Source)
2. Is Optifine installed or any other additional mods: **no** 

**Issue description**
Basically, I was playing the pack one day then the day of the update 2 days ago it just wouldn't let me. I played prior to that day and nothing had changed, at least that I am aware of. None of my friends on the server seem to be having this problem. Whenever I would press play it would load up the bar on the bottom of the launcher and once compleated nothing happened. I did that a few more times. closed the launcher tried again, nothing. Completely uninstalled the pack then when to reinstall is and now the bar will go to completion but still have the button icon for install, which I have tried many times. I have 7 gb of my 16 gb of ram like before but still nothing. I figure it's something to do with the second line of the error message but I'm not sure what to do with it.

**Comment by xJon** [comment]
Hi! Are you able to run other similar modpacks through the Technic Launcher? Also, you can always run The 1.12.2 Pack through [CurseForge](https://www.curseforge.com/minecraft/modpacks/the-1122-pack) :)

**Comment by Whattabeach** [comment]
> Hi! Are you able to run other similar modpacks through the Technic Launcher? Also, you can always run The 1.12.2 Pack through [CurseForge](https://www.curseforge.com/minecraft/modpacks/the-1122-pack) :)

I'm actually able to install and run it today for some reason. I'm not sure why that is but that doesn't matter too much. I have it installed on curseforge now too in case something happens again. Also at the time I was able to run a modpack called tekkit lite.

**Comment by xJon** [comment]
Glad you got it working!
